orca-core/src/main/java/com/netflix/spinnaker/orca/Task.java

```diff
@@ -18,11 +18,15 @@
 import com.netflix.spinnaker.orca.pipeline.model.Stage;
 
 import javax.annotation.Nonnull;
+import java.util.Collections;
+import java.util.List;
 
 public interface Task {
   @Nonnull TaskResult execute(@Nonnull Stage stage);
 
   default void onTimeout(@Nonnull Stage stage) {}
 
   default void onCancel(@Nonnull Stage stage) {}
+
+  default List<String> aliases() { return Collections.emptyList(); }
```

Comment:
General idea is that a `Task` could be renamed and moved between packages.

These changes would be reflected as `aliases`.